src/session/sidebar/chat_row.rs

```diff
@@ -439,6 +477,38 @@ fn stringify_message(message: Message) -> String {
     }
 }
 
+fn stringify_made_message_call(is_outgoing: bool, data: MessageCall) -> String {
```

Comment:
This one has only one caller. Should we get rid of this method?

Reply:
It seems to help keeping the code above simpler, so I think it's better keeping it here. Maybe you can add a simple doc comment so that it's a bit clearer on what it does.

Reply:
There is a comment now. Reopen if there is something missing.